style.css

```diff
@@ -104,6 +106,220 @@
   list-style: none;
 }
 
+/* work-section */
+
+.work-section {
+  background-color: #f7f7f9;
+  width: 100%;
+  padding: 30px;
+  padding-top: 100px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.work-container {
+  display: grid;
+  grid-template-columns: auto;
+  border-radius: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  align-items: center;
+  padding: 20px;
+  gap: 30px;
+  margin-bottom: 20px;
+}
+
+.cards {
+  display: flex;
+  height: 100%;
+  width: 100%;
+  flex-direction: column;
+}
+
+.snapshot-portofolio img {
+  width: 295px;
+  height: 220px;
+}
+
+.blocks {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 100%;
+}
+
+.txtWrapper {
+  width: 80%;
+}
+
+.project-title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  padding: 5px;
+}
+
+/* Frame 2 */
+
+.tframes {
+  display: flex;
+  gap: 10px;
+  padding: 3px 0;
+}
+
+.tframes h3 {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.frames-list {
+  display: flex;
+  gap: 10px;
+  list-style: none;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: rgb(122, 134, 154);
+}
+
+.frames-list img {
+  height: 10px;
+  width: 10px;
+  top: 12px;
+  border-radius: 0;
+}
+
+.project-text {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.tag-list {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  padding: 10px;
+  gap: 10px;
+}
+
+.tag-list li {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #6070ff;
+  background-color: rgb(235, 235, 255);
+  padding: 4px 12px;
+  border-radius: 8px;
+}
+
+.blocks-buttonWrapper {
+  padding: 5px 0;
+}
+
+.blocks-button {
+  border: 1px solid rgb(96, 112, 255);
+  border-radius: 8px;
+  padding: 12px;
+  color: #396df2;
+  background-color: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.blocks-button:hover {
+  background-color: #6070ff;
+  color: white;
+  transition: all ease 1s;
+  box-shadow: 0 -5px 10px 0 rgba(0, 0, 0, 0.5);
+}
+
+.blocks-button:active {
+  background-color: hsl(235, 72%, 48%);
+  color: white;
+  transition: all ease 1s;
+}
+
+/* About Myself-section */
+.about-section {
+  width: 100%;
+  height: auto;
+  padding-bottom: 150px;
+  padding-top: 100px;
+  background-image: url(img/about-img/about\ bg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-top-right-radius: 80px 80px;
+}
+
+.about-Container {
+  display: flex;
+  gap: 12px;
+  flex-direction: column;
+  width: 80%;
+  margin-left: 20px;
+}
+
+.skil-List {
+  display: flex;
+  flex-direction: column;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  padding-top: 12px;
+  list-style-type: none;
+  width: 90vw;
+  margin: 0 auto;
+}
```

Comment:
- [ ] As pointed out by previous reviewer, if you notice the right pointed arrows in front of the skill list, are aligned in a such a way that they are not beyond the borders of the languages. Kindly try as much as possible to align the nameList arrows to the left to match the Figma design ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/235312941-d10383d1-6cb2-4198-b469-79fc7c59109f.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/235312960-a8cbf006-2c1d-4b3a-b459-84e06737c772.png)